### ISSUE TITLE
Shutting off Via

### ISSUE THREAD

**Issue by BigBlueHat** [hypothesis/via#8]
Once you're in Via, all links go to other pages with Via enabled--which is great! :smile: 

However, there's a point at which you may want to "shut it off" / leave Via mode, etc.

Yes, this is already possible by editing the URL, but that's pretty geeky. :computer: 

Somewhere or other in the UI, we should have a button to load the non-Via-version. Not sure where to put that yet...but it's needed.

**Comment by tilgovi** [comment]
:+1:

**Comment by ikreymer** [comment]
Yep, definitely makes sense. Could this be part of the toolbar itself? Maybe it can be inited with a flag to indicate that its running under via, and then show an extra 'Exit Via' button that would just redirect to the live page?

**Comment by tilgovi** [comment]
I don't really want to add buttons to the toolbar that are nonsensical except in the presence of some other thing. However, the toolbar is extensible. `annotator.toolbar.items` can be added to.

**Comment by dwhly** [comment]
An option might be a thin header at the top (or bottom) across the whole page-- it's perhaps a familiar way that others have used to cancel proxy services.  i.e. w/ an X at one side that would cancel the frame and redirect to the live page.

I *thought* stumbleupon used to do this, but when I just checked they no longer appear to offer a way to remove the frame that way.

**Comment by tilgovi** [comment]
We should wait for feedback / user testing.
It may be that people are comfortable just opening a link in a new tab.

**Comment by ikreymer** [comment]
I think banner at top would work. To clarify, Via currently doesn't use iframes, it just inserts directly into the page (though frames are an option also -- banner can work either way).
